spec/system/candidate_interface/candidate_declines_offer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate declines an offer', sidekiq: true do
+  include CourseOptionHelpers
+
+  scenario 'Candidate views an offer and declines' do
+    given_i_am_signed_in
+    and_i_have_an_offer
+
+    when_i_visit_the_application_dashboard
+    then_i_see_the_view_and_respond_to_offer_link
+
+    when_i_click_on_view_and_respond_to_offer_link
+    and_i_decline_the_offer
+    and_i_confirm_the_decline
+
+    then_a_slack_notification_is_sent
```

Comment:
this reads a bit strange, 
maybe `then_a_slack_notification_is_sent`

Reply:
Fixed